src/universal/modules/meeting/components/MeetingCheckInPrompt/CheckInQuestion.js

```diff
@@ -100,22 +101,14 @@ class CheckInQuestion extends Component {
       ? 'Tap to customize the Social Check-in question.'
       : 'Upgrade to a Pro Account to customize the Social Check-in question.';
 
-    const tooltipProps = {
-      tip: <div>{tip}</div>,
-      originAnchor: {vertical: 'bottom', horizontal: 'center'},
-      targetAnchor: {vertical: 'top', horizontal: 'center'},
-      hideOnFocus: true
-    };
-
-    const tooltipHiddenProps = {
-      ...tooltipProps,
-      isOpen: false
-    };
-
-    const whichTooltipProps = (isEditing || !isFacilitating) ? tooltipHiddenProps : tooltipProps;
-
     return (
-      <Tooltip {...whichTooltipProps}>
+      <Tooltip
+        tip={<div>{tip}</div>}
+        originAnchor={{vertical: 'bottom', horizontal: 'center'}}
+        targetAnchor={{vertical: 'top', horizontal: 'center'}}
+        hideOnFocus
+        isOpen={isFacilitating && !isEditing}
```

Comment:
This means that the tooltip will always be open when facilitating?

Reply:
My hacky looking `whichTooltipProps` meant to enable on hover, except in the case that the facilitator is editing or the user is not the facilitator, where it forces it to be closed.

Reply:
ah shoot yeah i need a ternary, forgot the whole default logic